commons-app/pom.xml

```diff
@@ -282,6 +282,18 @@
 			<version>${mysql.jdbc.version}</version>
 		</dependency>
 
+		<dependency>
```

Comment:
@cmader, I know you mention this on the UV workshop, but why do we need to add the following two dependencies to commons-app?